test/tests/multiapps/command_line/master.i

```diff
@@ -18,6 +18,7 @@
     positions = '0 0 0
                  1 1 1'
     input_files = 'sub.i'
-    cli_args = 'Mesh/nx=10 Mesh/nx=100'
+    cli_args = 'Mesh/mesh/type=GeneratedMeshGenerator;Mesh/mesh/dim=1;Mesh/mesh/nx=10
```

Comment:
Should this be
```
Mesh/mesh/type=GeneratedMeshGenerator Mesh/mesh/dim=1 Mesh/mesh/nx=10;
Mesh/mesh/type=GeneratedMeshGenerator Mesh/mesh/dim=1 Mesh/mesh/nx=100
```

Reply:
I'm open to either but we need to decide. I m redoing the parsing of CLI argument strings

Reply:
old syntax with "space" to declare change of argument is more compatible with what we do elsewhere:
"space" for vector
; for vector of vector